performance-tests/scenarios/05_sustained_load.yml

```
@@ -0,0 +1,63 @@
+config:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)